apidocs/sundials.rst

```diff
@@ -0,0 +1,28 @@
+Sundials Solver Options
+#######################
+
+Selecting Precision
+-------------------
+Sundials can be built with different precisions, and scikits.odes supports this.
```

Comment:
I would suggest to add here what they are: single, double or extended, with double the default.